dataclassy/__init__.py

```diff
@@ -5,7 +5,7 @@
  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 """
 from .decorator import dataclass, make_dataclass
-from .dataclass import DataClass, Internal
+from .dataclass import DataClass, DataClassMeta, Internal
```

Comment:
I don't like this; I'd prefer to demonstrate that `DataClassMeta` is more internal to give us the freedom to tweak its implementation over time (using it externally is somewhat "at your own risk"). This is [noted](https://github.com/biqqles/dataclassy#meta) in the docs.

Reply:
sure, done